Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```diff
@@ -0,0 +1,1891 @@
+      isArray: false
+      name: position
+      predefined:
+      - pre
+      - post
+    - auto: PREDEFINED
+      default: false
+      description: Rule action.
+      isArray: false
+      name: action
+      predefined:
+      - deny
+      - allow
+      - drop
+      - reset-client
+      - reset-server
+      - reset-both
+    - default: false
+      description: The security rule's description.
+      isArray: false
+      name: description
+      required: false
+    - default: false
+      description: Rule log setting.
+      isArray: false
+      name: log_setting
+      required: false
+    - default: false
+      description: A comma-separated list of applications.
+      isArray: false
+      name: application
+      required: false
+    - default: false
+      description: A comma-separated list of categories.
+      isArray: true
+      name: category
+      required: false
+    - default: false
+      description: A comma-separated list of destination networks.
+      isArray: false
+      name: destination
+      required: false
+    - default: false
+      description: A comma-separated list of destination HIPs.
+      isArray: false
+      name: destination_hip
+      required: false
+    - default: false
+      description: Security profiles to apply to the traffic.
+      isArray: false
+      name: profile_setting
+      required: false
+    - default: false
+      description: Services the rule applies to.
+      isArray: false
+      name: service
+      required: false
+    - default: false
+      description: A comma-separated list of source networks.
+      isArray: true
+      name: source
+      required: false
+    - default: false
+      description: A comma-separated list of source HIPs.
+      isArray: false
+      name: source_hip
+      required: false
+    - default: false
+      description: A semi-colon (;) separated list of source user(s).
+      isArray: false
+      name: source_user
+      required: false
+    - default: false
+      description: A comma-separated list of rule tags.
+      isArray: true
+      name: tag
+      required: false
+    - default: false
+      description: A comma-separated list of source zones.
+      isArray: true
+      name: from
+      required: false
+    - default: false
+      description: A comma-separated list of destination zones.
+      isArray: true
+      name: to
+      required: false
+    - default: false
+      description: Whether the rule is disabled.
+      isArray: false
+      name: disabled
+      required: false
+    - default: false
+      description: Negate source
+      isArray: false
+      name: negate_source
+      required: false
+    - default: false
+      description: Negate the destination.
+      isArray: false
+      name: negate_destination
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: This argument specifies rather to append or overwrite the....
```

Comment:
This is a true/false result.
So it needs to be something like:
"Whether to overwrite the xxx."
And you need to fill in the xxx.